prog-o-meter.py

```diff
@@ -168,15 +169,9 @@ def input_buttons(self):
         """Display the buttons on the canvas.
         
         Dispays a set of two radio buttons, for the user to indicate whether they are a new or returning user. 

```

Comment:
There is a typo at "Displays".

Reply:
Haha, you and @garroadran are really good at catching those. I haven't really thought about proofreading my comments in the past, but it is definitely something I will start doing. Thanks for the catch.
I will fix the typo. Will you check if the code can be approved either, or add any further feedback you might have :)